tensorboard/webapp/metrics/internal_types.ts

```diff
@@ -14,10 +14,14 @@ limitations under the License.
 ==============================================================================*/
 import {HistogramMode} from '../widgets/histogram/histogram_types';
 
-import {PluginType} from './data_source/types';
-
 export {HistogramMode};
 
+export enum PluginType {
```

Comment:
Just checking, was there a reason why the `PluginType` definition moved from data_source/types to internal_types?  I don't see anything in this PR that requires it to move.

Reply:
It creates a circular dep before.

- InternalTypes requires to read PluginType from `data_source/types`
- `data_source/types` require to read `TooltipSort` enum from `InternalTypes`.